op-deployer/book/src/reference-guide/artifacts-locators.md

```diff
@@ -0,0 +1,32 @@
+# Artifacts Locators
+
+OP Deployer calls into precompiled contract artifacts. To make this work, OP Deployer uses artifacts locators to
```

Comment:
Might be good to indicate "precompiled contract artifacts" includes "forge solidity scripts + op chain contracts"